### DISCUSSION TITLE
JST-EH connector does not fit on S21 on Emura 3 board

### DISCUSSION THREAD

**Discussion by migueldc73** [revk/xESP32-Faikout#272] [Q&A]
So, I bought the Faikin module, got it working, but can't seem to find a way to connect it to my S21 connector on an Emura 3
I bought a few JST-EH connectors based on the instructions

> The S21 plug needed is [JST, EH Female Connector Housing, 2.5mm Pitch, 5 Way, 1 Row](https://uk.rs-online.com/web/p/wire-housings-plugs/3116237), which needs pins and crimps as well.

The JST-EH is perfect for the Faikin side, but not for the S21 connector on the Emura board, which is clearly smaller than the JST-EH, with a pitch of probably 1.5 or 2mm.
![315925198-59e96ec8-1743-4a68-a146-783db5359895](https://github.com/revk/ESP32-Faikin/assets/107763234/9b9f2cbb-d241-4328-9544-3dcb1bf737ad)

As I can't find any EKRS21 cables in stock for shipping less than 4-5 weeks in Belgium/Netherlands, I am desperate to find the right connector model nr, so I can order it and make my own cable.
From another discussion thread I presume, it is the PAP-05V-S connector, but would like to have some re-assurance before I buy another set of useless connectors.

**Comment by wurmtal868** [comment]
The S21 seams to be a "JST PAP-05V-S" connector, which are not very common. 
I bought some ready made Chinese 2mm pitch JST cables and added 2,5 mm Dupont connectors for the Faikin side.
The Chinese 2mm JST are a little too thick but a little persuasion with a cutter knife made them fit. ;-) 

As reference here is another thread about the cables: https://github.com/revk/ESP32-Faikin/discussions/260

**Reply by 44616E** [reply]
It is a JST PAP-05V-S I ordered them and the crimp terminals from digikey and it fits perfectly.

**Reply by migueldc73** [reply]
perfect, will give it a try !

**Comment by PianSom** [comment]
I got this to work with an official Daikin EKRS21 adapter plus some F-F Dupont cables.

Reading now the HA forums, I have seen this[ ](https://www.aliexpress.com/item/1005005562174022.html)cheap Aliexpress cable which I would try if I was starting again.

**Comment by PianSom** [comment]
Very small link above is to https://www.aliexpress.com/item/1005005562174022.html
